app/assets/javascripts/discourse/app/components/composer-body.js

```diff
@@ -208,6 +224,10 @@ export default Component.extend(KeyEnterEscape, {
 
     this.setupComposerResizeEvents();
 
+    this.updateHeightOnViewportResize();
```

Comment:
Do we need to call this directly? I suspect both the composer's height and the CSS custom property would already be set on initialization, so this seems redundant.